mempool/clist_mempool.go

```diff
@@ -0,0 +1,780 @@
+package mempool
```

Comment:
Maybe add do.go file documenting where this is coming from, why it was copied and what was modified.

Reply:
Resolved in 6d91a8c.